ApiResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace APIAutomation
+{
+    public class ApiResponse
```

Comment:
Where do you use this class in your tests?

Reply:
Ok, it was deleted